src/main/java/org/dcsa/ebl/model/transferobjects/TransportDocumentTO.java

```diff
@@ -1,57 +1,67 @@
 package org.dcsa.ebl.model.transferobjects;
 
+import com.fasterxml.jackson.annotation.JsonFormat;
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.Data;
-import lombok.EqualsAndHashCode;
 import lombok.NoArgsConstructor;
 import org.dcsa.core.events.edocumentation.model.transferobject.CarrierClauseTO;
-import org.dcsa.core.events.model.Location;
+import org.dcsa.core.events.edocumentation.model.transferobject.ChargeTO;
+import org.dcsa.core.events.model.enums.CarrierCodeListProvider;
+import org.dcsa.core.events.model.transferobjects.LocationTO;
 import org.dcsa.core.events.model.transferobjects.ShippingInstructionTO;
-import org.dcsa.ebl.model.TransportPlan;
-import org.dcsa.ebl.model.base.AbstractTransportDocument;
-import org.dcsa.ebl.model.enums.ServiceType;
-import org.dcsa.ebl.model.enums.ShipmentTerm;
 
 import javax.validation.Valid;
 import javax.validation.constraints.Size;
+import java.time.LocalDate;
+import java.time.OffsetDateTime;
 import java.util.List;
 
 @Data
 @NoArgsConstructor
-@EqualsAndHashCode(callSuper = true)
-public class TransportDocumentTO extends AbstractTransportDocument {
+public class TransportDocumentTO {
 
-    @Valid
-    @JsonProperty("placeOfIssue")
-    private Location placeOfIssueLocation;
+  @Size(max = 20)
+  private String transportDocumentReference;
 
-    @Valid
-    private ShipmentTerm shipmentTermAtOrigin;
+  private OffsetDateTime transportDocumentCreatedDateTime;
 
-    @Valid
-    private ShipmentTerm shipmentTermAtDestination;
+  private OffsetDateTime transportDocumentUpdatedDateTime;
 
-    @Valid
-    private ServiceType serviceTypeAtOrigin;
+  @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
+  private LocalDate issueDate;
 
-    @Valid
-    private ServiceType serviceTypeAtDestination;
+  @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
+  private LocalDate shippedOnBoardDate;
```

Comment:
onboard is one word

Reply:
The field name is aligned with the Open API specification. So won't fix until Open API spec is changed